src/Forms/Prism.Forms/Navigation/INavigationParameters.cs

```diff
@@ -2,20 +2,57 @@
 
 namespace Prism.Navigation
 {
+    /// <summary>
+    /// Provides a way for the INavigationService to pass parameteres during navigation.
```

Comment:
INavigationService -> `<see cref="INavigationService" />`

Reply:
There is no need for docs on anything in the body as we're removing the contents to a common IParameters